src/backend/vulkan/src/command.rs

```diff
@@ -337,20 +441,27 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
                     },
                     image_extent: dst.extent.clone(),
                 }
-            }).collect();
+            })
+            .collect();
 
         unsafe {
             self.device.0.cmd_copy_buffer_to_image(
-                self.raw, // commandBuffer
-                src.raw, // srcBuffer
-                dst.raw, // dstImage
+                self.raw,                       // commandBuffer
```

Comment:
I don't think we need the comments here, it's mostly self-describing.
For non-obvious things, we could add local variables, e.g. `let command_buffer = self.raw;`